qiskit/synthesis/clifford/clifford_decompose_layers.py

```diff
@@ -135,10 +135,23 @@ def synth_clifford_layers(
     )
 
     layeredCircuit.append(S2_circ, qubit_list)
-    layeredCircuit.append(CZ2_circ, qubit_list)
 
-    CXinv = CX_circ.copy().inverse()
-    layeredCircuit.append(CXinv, qubit_list)
+################################################################################
```

Comment:
you may delete this line (and also the similar lines of `###` of the changed parts)